site/en/docs/webstore/i18n/index.md

```diff
@@ -1,132 +1,124 @@
-title: Internationalizing Your App
+title: Internationalize your extension
 date: 2017-08-30
-description: How to internationalize your Chrome Web Store app.
+updated: 2021-10-29
+description: How to internationalize your Chrome Web Store extension.
 ---
 
-Unless your app is strictly local in scope, you should _internationalize_ it, making it easy to
+This page describes how to support multiple languages of an extension in the Chrome Web Store.
+
+Unless your extension is strictly local in scope, you should _internationalize_ it, making it easy to
 adapt to various languages and regions. You can then _localize_ it—translate and otherwise adapt it
 so that it works well in a particular locale.
 
-You can internationalize your app even if it initially supports just one locale. For example, you
-might initially publish your app in English (locale code: "en"). Then, after a few weeks or months,
-you might add support for additional locales such as French (locale code: "fr") and Arabic (locale
-code: "ar").
-
 {% Aside %}
 The word _internationalization_ is often abbreviated to _i18n_, referring to the 18
 letters between the letters _i_ and _n_. _Localization_ is sometimes abbreviated to _l10n_.
 {% endAside %}
 
 ## Overview
 
-Here are the steps you need to follow when you internationalize and localize an app to be published
+You can follow these steps to internationalize and localize an extension to be published
 in the Chrome Web Store:
 
-1.  Decide which [locales][1] you want to support.
-2.  Internationalize and localize the app's UI.
-3.  Localize your app's [name and description][2].
-4.  Upload your app using the Chrome Developer Dashboard.
-5.  Edit your listing, providing localized versions of the [detailed description][3] and
-    [screenshots][4].
-6.  [Specify which regions][5] you want to support.
-
-How you accomplish step 2 depends on whether you're writing an extension, packaged app, or hosted
-app. [Extensions][6] and [packaged apps][7] can use the [extension system i18n support][8] to
-provide the appropriate UI strings for each locale. [Hosted apps][9] don't have access to the
-extension i18n support for their main UIs, but they can use whatever other i18n techniques and
-libraries suit the app.
-
-In step 3 you localize the name and description by editing the manifest file and providing strings
-for your chosen locales. Once you've uploaded your app (step 4), you can finish your app's listing
-(step 5).
+1. Decide which [locales][supported-locales] you want to support.
+1. Internationalize and localize the extension's UI using the [extension system i18n support][i18n].
+1. Localize your extension's [name and description][manifest].
+1. [Upload your extension][publish] using the Chrome Developer Dashboard.
+1. Edit your listing, providing localized versions of the [detailed description][cws-description], a YouTube video URL and [screenshots][cws-screenshots].
+1. Specify which [regions][regions] you want to support.
```

Comment:
```suggestion
1. Decide which [locales][supported-locales] you want to support.
1. Internationalize and localize the extension's UI using the [extension platform's i18n
   support][i18n].
1. Localize your extension's [name and description][manifest].
1. [Upload your extension][publish] using the Chrome Developer Dashboard.
1. Edit your listing, providing localized versions of the [detailed description][cws-description], a
   YouTube video URL and [screenshots][cws-screenshots].
1. Specify which [regions][regions] you want to support.
```

We tend to refer to the "extension platform" rather than system. Also , wrap at 100 characters.